pkg/build/builder/util.go

```diff
@@ -14,11 +11,7 @@ type Builder interface {
 // imageTag returns the tag to be used for the build. If a registry has been
 // specified, it will prepend the registry to the name
 func imageTag(build *api.Build) string {
-	tag := build.Parameters.Output.ImageTag
-	if !strings.HasPrefix(tag, build.Parameters.Output.Registry) {
-		tag = fmt.Sprintf("%s/%s", build.Parameters.Output.Registry, tag)
-	}
-	return tag
+	return build.Parameters.Output.DockerImageReference
```

Comment:
I don't see any value of leaving this method as is, I'd rather like to get rid of it.